### PR TITLE
Update releasing script to allow specifying suffixes for different packages

### PR DESCRIPTION
**Pull request by aabmass**
- Remove pinning stuff, as its no longer needed
- Can now specify alternate suffixes for certain packages. This is in line with what our ops-java repo does, where metrics and resource detector are marked alpha, but otherwise the same version. See updated `releasing.md` for explanation
- Verify everything is PEP440/pypi compliant now

Release PR generated from these changes here: https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/pull/135

### CODE REVIEW

**Comment by punya** [comment]
How about having a config file that specifies which packages get the alternate suffix and which don't? Otherwise, someone has to construct that long command line each time.

**Comment by aabmass** [comment]
> How about having a config file that specifies which packages get the alternate suffix and which don't? Otherwise, someone has to construct that long command line each time.

Let me give it a shot to see how it looks

File: docs/releasing.md [hunk 1]
```diff
@@ -95,23 +109,17 @@ buggy) small script for the example PR's tag:
...
 ```bash
 # pull in the new release tag
 git fetch origin
 
-# create branch and move stable
-git branch release/0.11b0 v0.11b0
+# move stable
 git tag -d stable
 git push origin :refs/tags/stable
-git tag stable v0.11b0
```
Comment (aabmass) [inline docs/releasing.md:110]: add back

**Comment by punya** [comment]
It could be as simple as a map hardcoded in `release.py`.

**Comment by aabmass** [comment]
Thanks @punya it looks much better now

File: docs/releasing.md [hunk 2]
```diff
@@ -41,17 +41,23 @@ branch. The script does the following:
...
 ```bash
 ./release.py \
     --release_version 0.11b1 \
-    --new_dev_version 0.12.dev0 \
-    --ot_version "==0.11b0"
+    --new_dev_version 0.12.dev0
 ```
 
+You can also specify alternative suffixes to add for certain packages by
+updated the `ALTERNATE_SUFFIXES` map in `release.py`. This lets you mark some
```
Comment (punya) [inline docs/releasing.md:54]: *updating

**Review by punya — APPROVED** [review]

**Pull request merged**